graphql/jsonw.go

```
@@ -81,3 +82,10 @@ func lit(b []byte) Marshaler {
 		w.Write(b)
 	})
 }
+
+// MarshalJSON marshals an interface into JSON.
+func MarshalJSON(i interface{}) Marshaler {
+	return WriterFunc(func(w io.Writer) {
+		json.NewEncoder(w).Encode(i)
```

There's an error being ignored here, I wonder why gometalinter isn't complaining.